src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1608,7 +1609,7 @@ private function get_enqueued_attribute_value( $comparable_name ) {
 	 *     $p->get_attribute( 'enabled' ) === true;
 	 *     $p->get_attribute( 'aria-label' ) === null;
 	 *
-	 *     $p->next_tag( array() ) === false;
+	 *     $p->next_tag() === false;
```

Comment:
I also adjusted these, explicitly passing an empty array doesn't feel necessary

Reply:
👍 and there's PHPUnit tests that just call `next_tag()` without passing an empty array.